packages/amazonq/src/extension.ts

```diff
@@ -97,10 +97,8 @@ export async function activateAmazonQCommon(context: vscode.ExtensionContext, is
     globals.manifestPaths.endpoints = context.asAbsolutePath(join('resources', 'endpoints.json'))
     globals.regionProvider = RegionProvider.fromEndpointsProvider(makeEndpointsProvider())
 
-    const qOutputChannel = vscode.window.createOutputChannel('Amazon Q', { log: true })
-    const qLogChannel = vscode.window.createOutputChannel('Amazon Q Logs', { log: true })
-    await activateLogger(context, amazonQContextPrefix, qOutputChannel, qLogChannel)
-    globals.outputChannel = qOutputChannel
+    const qLogChannel = vscode.window.createOutputChannel('Amazon Q', { log: true })
```

Comment:
Let's keep "Logs" for now. That aligns with Toolkit, and also signals that these are potentially low-level messages. And it keeps the door open for potentially in the future having higher-level messages in a `Amazon Q` channel

Reply:
Sounds good, changed